src/DssSpell.sol

```diff
@@ -39,13 +43,71 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    address immutable internal ESM            = DssExecLib.getChangelogAddress("MCD_ESM");
-    address immutable internal RWA_LIQ_ORACLE = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+    uint256 internal constant FOUR_NINE_PCT_RATE = 1000000001516911765932351183;
+
+    uint256 internal constant MILLION = 10**6;
+    uint256 internal constant RAD     = 10**45;
 
     function actions() public override {
 
-        // ----- Additional ESM authorization fix -----
-        DssExecLib.authorize(RWA_LIQ_ORACLE, ESM);
+        uint256 Art;
+        uint256 rate;
+        uint256 line;
+        uint256 sumLines;
+        uint256 sumDebts;
+        VatLike vat = VatLike(DssExecLib.vat());
+
+        // ---------------- RWA008-A Off-boarding Phase 0 ----------------
+        // Forum: https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605/51
+
+        // Set RWA008-A Debt Ceiling to 0:
+        (Art, rate, , line, ) = vat.ilks("RWA008-A");
+        DssExecLib.setIlkDebtCeiling("RWA008-A", 0);
+        sumLines += line;
```

Comment:
So a way to do this (which is more in line with previous patterns) is that we use `lineReduction` which increases by line and then decreases by Art * rate.

Functionally this is not really different from what you're doing here but just a heads up.

Reply:
Done.